helm/01-config-helm/solve-kubernetes-vm

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
All these empty ones can just be deleted, they serve no purpose for the functionality